r/R/dataset.R

```diff
@@ -360,6 +362,8 @@ IpcFileFormat <- R6Class("IpcFileFormat", inherit = FileFormat)
 #' - `$UseThreads(threads)`: logical: should the scan use multithreading?
 #' The method's default input is `TRUE`, but you must call the method to enable
 #' multithreading because the scanner default is `FALSE`.
+#' - `$BatchSize(batch_size)`: integer: Maximum row count of scanned record
```

Comment:
Probably should make clear that this _sets_ the batch size. Also would be nice to know why you might want to do this.

Reply:
I'll add to the comment